data/avgi/avgi outfits.txt

```diff
@@ -413,7 +413,6 @@ outfit "Remass Injector"
 	thumbnail "outfit/remass injector"
 	"mass" 6
 	"outfit space" -6
-	"engine capacity" -3
```

Comment:
3 engine capacity already isn't much, especially for this much thrust. I think if the remass injector is too weak, the thing to do would be tone down the fuel usage slightly rather than have it not take up engine space.